inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -99,15 +67,7 @@ export async function loadProject(args: {
 		)
 	}
 
-	let fs: Repository["nodeishFs"]
-	if (args.nodeishFs) {
-		// TODO: deprecate
-		fs = args.nodeishFs
-	} else if (args.repo) {
-		fs = args.repo.nodeishFs
-	} else {
-		throw new LoadProjectInvalidArgument(`Repo missing from arguments.`, { argument: "repo" })
-	}
+	const fs = args.repo.nodeishFs
 
 	const nodeishFs = createNodeishFsWithAbsolutePaths({
```

Comment:
@samuelstroschein I think you're asking to remove the tryCatch on L88.

Reply:
nope this one https://github.com/opral/monorepo/issues/2069

Reply:
https://discord.com/channels/@me/1196510316717154314/1198999268531380225

Reply:
aah - ok - 🙏 👀